pkg/cluster/deploystorage.go

```diff
@@ -152,43 +147,11 @@ func (m *manager) deployStorageTemplate(ctx context.Context, installConfig *inst
 		Schema:         "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
 		ContentVersion: "1.0.0.0",
 		Resources: []*arm.Resource{
-			rbac.ResourceGroupRoleAssignmentWithName(
-				rbac.RoleContributor,
-				"'"+clusterSPObjectID+"'",
-				"guid(resourceGroup().id, 'SP / Contributor')",
-			),
```

Comment:
nit: function replacing this resource `clusterServicePrincipalRBAC(clusterSPObjectID)` is at the bottom of the slice but does not seem to be a big issue.